ex6.2_timer.py

```diff
@@ -2,6 +2,13 @@
 
 
 def timer(f, *args, **kwargs):
+    """
+    function that gets parameters and calculate the time that takes for it.
+    :param f: function
```

Comment:
Don't use abbreviations in variable names -
https://softwareengineering.stackexchange.com/questions/72593/how-to-abbreviate-variable-names